app/src/main/java/com/workingagenda/democracydroid/MediaService.java

```diff
@@ -88,16 +58,18 @@ public IBinder onBind(Intent intent) {
     public void onCreate() {
         Log.d("MediaService", "OnCreate");
 
-        TrackSelection.Factory videoFactory = new AdaptiveTrackSelection.Factory();
+        ExoTrackSelection.Factory videoFactory = new AdaptiveTrackSelection.Factory();
 
-        TrackSelector trackSelector = new DefaultTrackSelector(videoFactory);
+        TrackSelector trackSelector = new DefaultTrackSelector(getApplicationContext(), videoFactory);
         LoadControl loadControl = new DefaultLoadControl();
 
         // Create Player
         player = new SimpleExoPlayer.Builder(getApplicationContext())
-                .setLoadControl(loadControl).setTrackSelector(trackSelector).build();
+                .setLoadControl(loadControl)
+                .setTrackSelector(trackSelector)
+                .build();
 
-        player.setPlayWhenReady(true);
```

Comment:
How come you changed this to `play`?

Reply:
https://github.com/google/ExoPlayer/blob/release-v2/RELEASENOTES.md#2120-2020-09-11

> Add play and pause convenience methods. They are equivalent to setPlayWhenReady(true) and setPlayWhenReady(false) respectively.